examples/1d/plot_filters.py

```diff
@@ -1,164 +1,143 @@
+# -*- coding: utf-8 -*-
+"""plot_filters.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1x0VXqMpE2-bmwAwPO78UHkwKJeDHQnF2
 """
-Plot the 1D wavelet filters
-===========================
+
+# Commented out IPython magic to ensure Python compatibility.
+!pip install git+https://github.com/kymatio/kymatio.git@dev 
+# %matplotlib inline
+
+"""# Plot the 1D wavelet filters
 Let us examine the wavelet filters used by kymatio to calculate 1D scattering
 transforms. Filters are generated using the
 :meth:`kymatio.scattering1d.filter_bank.scattering_filter_factory` method,
 which creates both the first- and second-order filter banks.
-"""
 
-###############################################################################
-# Preliminaries
-# -------------
-# First, we import the `scattering_filter_factory` method, which we will use
-# to generate the filters.
+## Preliminaries
```

Comment:
Again, it would be good to keep the original style of the comments to reduce the diff and preserve the same style across examples.